apps/publisher/themes/default/js/list_assets.js

```diff
@@ -189,7 +189,10 @@ var createQuery = function(options) {
     var input = $('#inp_searchAsset').val();
     var category = options.category || undefined;
     var searchQueryString = '?';
-    input = setDefaultSearchQuery(input);
+    input = input.trim();
+    if (input.indexOf(":") == -1 && input.trim() !== "") {
```

Comment:
Thanks for pointing it out. Fixed with the following PR - https://github.com/wso2/carbon-store/pull/833